SugarRecord/Source/Foundation/Entities/Request.swift

```diff
@@ -45,7 +45,17 @@ public struct FetchRequest<T: Entity>: Equatable {
             .request(withPredicate: NSPredicate(format: "\(key) == %@", value))
     }
     
-    public func sortedWith(sortDescriptor: NSSortDescriptor) -> FetchRequest<T> {
+    public func filteredWith(_ key: String, equalTo value: [String]) -> FetchRequest<T> {
```

Comment:
What about renaming it to:

```swift
public func filtered(with key: String, in value: [String]) ->FetchRequest<T>
```
That way you can call it like

```swift
request.filtered(with: "name", in: ["Pedro", "Maria"])
```

I think it's more readable.